Frontend/src/shared/services/axiosInstance.js

```diff
@@ -15,4 +15,14 @@ axiosInstance.interceptors.request.use((config) => {
   return config;
 });
 
+axiosInstance.interceptors.response.use((response) => response, (error) => {
+  if (error?.response?.status === 440) {
+    localStorage.removeItem('user');
```

Comment:
Maybe this could be some kind of global `logout()` function? It would also make sense to `ProfileService.clearProfileCache();` on logout, then this would be both in one.